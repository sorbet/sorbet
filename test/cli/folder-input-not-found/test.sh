#!/bin/bash

main/sorbet --silence-dev-message --dir test/cli/folder-input-not-found/files 2>&1
