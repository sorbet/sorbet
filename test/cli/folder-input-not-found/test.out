Directory `test/cli/folder-input-not-found/files` not found
